vNext/shared/AppInsightsCore/package.json

```diff
@@ -15,7 +15,6 @@
     "main": "dist/applicationinsights-core-js.js",
     "module": "dist-esm/applicationinsights-core-js.js",
     "types": "types/applicationinsights-core-js.d.ts",
-    "browser": "browser/applicationinsights-core-js.min.js",
```

Comment:
Is this not used anymore?

Reply:
forgot to doc this change. In debugger it is using minified version for only core due to this. Other packages' entry point is `"main": dist/<unminified-entry>.js`